src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h

```diff
@@ -36,6 +36,7 @@
 
 #include <OpenMS/CHEMISTRY/AASequence.h>
 #include <OpenMS/OPENSWATHALGO/DATAACCESS/DataStructures.h>
+#include <OpenMS/FILTERING/DATAREDUCTION/IsotopeDistributionCache.h>
```

Comment:
no need to include the header here.
Just forward declare should be enough (to save on compile time).
`class IsotopeDistributionCache;` within the OpenMS namespace.